### PR TITLE
Add recipe for tagmaps package (v0.17.6)

### PR DESCRIPTION
**Pull request by Sieboldianus**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

This is a pure python package. Many thanks for review in advance!

### CODE REVIEW

File: recipes/tagmaps/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+python:
```
Comment (marcelotrevisani) [inline recipes/tagmaps/conda_build_config.yaml]: Please remove this file, this is not necessary for your case

File: recipes/tagmaps/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  description: |
+    Perform tag clustering from spatially referenced and tagged records.
+    TagMaps performs itemized or global clustering based on a list of records with
+    multiple tags (or emoji) attached. Utilizes HDBSCAN for determining cluster
+    results at specific (optional user defined) distance. Tags, Locations and
+    Emoji are processed in descending order of global occurence, alpha
+    shapes are generated for cluster point clouds. Produces two output
+    shapefiles with cluster shapes that can be visualized, e.g. in ESRI ArcGIS.
+  doc_url: 
+  dev_url:
```
Comment (marcelotrevisani) [inline recipes/tagmaps/meta.yaml]: If you are not populating the ``doc_url`` and ``dev_url``, please remove then

**Comment by Sieboldianus** [comment]
@marcelotrevisani Many thanks for the hints. I solved these issues. Is the recipe ready to go? @conda-forge/staged-recipes

File: recipes/tagmaps/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - tagmaps = tagmaps.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - python>=3.5
```
Comment (chrisburr) [inline recipes/tagmaps/meta.yaml]: There needs to be a space before the version constraint else creating the feedstock will fail:
```suggestion
    - python >=3.5
```
This also applies to the run dependencies.

Comment (Sieboldianus) [reply recipes/tagmaps/meta.yaml]: Great, thanks for the commit!

**Pull request merged**